### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tiserett**
- [DEMO LINK](https://tiserett.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://tiserett.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <h1 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+
+      <h2 class="card__code">
+        Код товара: 195434
+      </h2>
```
Comment (AndriiRut) [inline src/index.html:22]:
```suggestion
      <p class="card__code">
        Код товара: 195434
      </p>
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+      <div class="card__rating">
+        <ul class="stars">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <div class="card__reviews">Отзывов: 5</div>
```
Comment (AndriiRut) [inline src/index.html:33]: checklist about semantic tags!

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: #060b35;
+}
+
+h1,
+h2,
+body {
+  margin: 0;
```
Comment (AndriiRut) [inline src/style.css:12]: Goole `how selector * works` (use MDN) to find out why you can add `margin: 0` to that selector and delete all `margin: 0` below and above =)

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__title {
+  margin: 40px 0 0;
```
Comment (AndriiRut) [inline src/style.css:41]:
```suggestion
  margin-top: 40px;
```

**Review by AndriiRut — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (tiserett) [reply src/index.html:22]: replaced all h2 tags with p

File: src/style.css (hunk 3 above)
Comment (tiserett) [reply src/style.css:12]: I heard that I should not add nothing to this selector but box-sizing property

File: src/style.css (hunk 4 above)
Comment (tiserett) [reply src/style.css:41]: changed all margins according to your comment

File: src/index.html (hunk 2 above)
Comment (tiserett) [reply src/index.html:33]: replaced all divs with sections and articles

Comment (AndriiRut) [reply src/index.html]: Actually, you could use:
`<div>
  <p>text</p>
</div>`

File: src/style.css (hunk 3 above)
Comment (AndriiRut) [reply src/style.css]: Box sizing and reset margin to 0 is ok!
this selector is very strong and resource-intensive, so you can use only such light styles with it.
Avoid to use `display: block` || `font-family: "Robot"` || `align-items` || `hight`.
But it's really ok to use it for the reset! =)

**Review by AndriiRut — CHANGES_REQUESTED** [review]
read this topic place and take the decision what to do witch `*` and `h1` & `p` * `ul`
[specific](https://www.w3schools.com/css/css_specificity.asp)

image

File: src/style.css (hunk 3 above)
Comment (tiserett) [reply src/style.css]: ok, i got it, gonna change my code right now))

**Review by AndriiRut — APPROVED** [review]
Good job!

**Review by kostya-kosyuk — CHANGES_REQUESTED** [review]

File: src/styles/reset.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  font-size: 12px;
+  color: #060b35;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
```
[the remark this answers is not in the document]
Comment (tiserett) [reply src/styles/reset.css:14]: почитай что писал Андрей в этом пул реквесте

File: src/styles/reset.css [hunk 6]
```diff
@@ -0,0 +1,15 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```
[the remark this answers is not in the document]
Comment (tiserett) [reply src/styles/reset.css:1]: нету какого-то определённого правила куда помещать import

**Pull request closed without merging**